product_docs/docs/pgd/5/cli/command_ref/assess/index.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: pgd assess
+navTitle: Assess
+---
+
+## Synopsis
+
+The `pgd assess` commands are used to assess the suitability of a Postgres server instance for migration to the EDB Postgres Distributed cluster.
+
+The command must be run with a DSN that connects to the Postgres server instance that you want to assess. The command will check the Postgres server instance for compatibility with the EDB Postgres Distributed cluster, and will provide a report on the compatibility of the Postgres server instance.
+
+## Options
+
+The assess command has no command specific options.
+
+See also [Global Options](/pgd/latest/cli/command_ref/#global-options).
+
+## Example
+
+```bash
+pgd assess
```

Comment:
add the `-o markdown` flag for report output. By default it gives tabular output